internal/statetransition/state_transition.go

```diff
@@ -2423,6 +2424,14 @@ func (a *Accumulator) SequentialDelta(
 	return uint32(maxReports), newCtx, transfers, hashPairs
 }
 
+func getMapKeys[K comparable, V any](m map[K]V) map[K]struct{} {
```

Comment:
Looks like you're turning a map into a set, maybe `toSet` would this a bit clearer? 

Are you planning on using this helper more in the future? (noticed it's only used once so far)

Reply:
we will probably need something like this in the future but for now it probably doesn't make sense to have a generic